views/js/uiForm.js

```diff
@@ -728,28 +728,38 @@
              */
             function showPropertyListValues() {
                 const $this = $(this);
-                const elt = $this.parent("div");
+                const elt = $this.parent('div');
                 let classUri;
 
                 //load the instances and display them (the list items)
-                $(elt).parent("div").children("ul.form-elt-list").remove();
+                $(elt).parent('div').children('ul.form-elt-list').remove();
                 classUri = $this.val();
+
                 if (classUri && classUri.trim()) {
-                    $this.parent("div").children("div.form-error").remove();
+                    $this.parent('div').children('div.form-error').remove();
+
                     $.ajax({
                         url: context.root_url + 'taoBackOffice/Lists/getListElements',
-                        type: "POST",
-                        data: {listUri: classUri},
-                        dataType: 'json',
+                        type: 'GET',
+                        data: {
+                            listUri: classUri,
+                        },
                         success: function (response) {
-                            let html = "<ul class='form-elt-list'>",
+                            let html = '<ul class="form-elt-list">',
                                 property;
-                            for (property in response) {
-                                if(!response.hasOwnProperty(property)) {
+
+                            for (property in response.data.elements) {
+                                if (!response.data.elements.hasOwnProperty(property)) {
                                     continue;
                                 }
```

Comment:
Suggestion: Maybe it would be slightly better to use `Object.prototype.hasOwnProperty.call()` instead of calling the method directly on the object instance (ESLint would probbaly complain about this). See https://dev.to/aman_singh/what-s-the-deal-with-object-prototype-hasownproperty-call-4mbj,  https://stackoverflow.com/questions/12017693 & https://eslint.org/docs/rules/no-prototype-builtins

```suggestion
                                if (!Object.prototype.hasOwnProperty.call(response.data.elements, property)) {
                                    continue;
                                }
```

Reply:
Thanks for your suggestion 🙂
@bziondik @adriloma , can you please check this?

Reply:
yes, it is correct